utils/test-utils/src/main/java/com/microsoft/device/dualscreen/utils/test/SurfaceDuoUtils.kt

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.dualscreen.utils.test
```

Comment:
This class is very `Surface Duo` specific do we need that? We should have testing utilities (if needed) generic valid for any foldable device.